module/VuFind/src/VuFind/AjaxHandler/GetItemStatuses.php

```diff
@@ -451,6 +451,7 @@ protected function renderFullStatus($record, $simpleStatus, array $values = [])
                 'statusItems' => $record,
                 'simpleStatus' => $simpleStatus,
                 'callnumberHandler' => $this->getCallnumberHandler(),
+                'holdingsTextFieldNames' => $this->ils->getHoldingsTextFieldNames(),
```

Comment:
I think it would be a good idea to make this configurable via config.ini. There are a few possible approaches, depending on how granular we want this to be -- e.g. it could just be something like `include_holdings_text_fields_in_full_status = true|false` in the `[Item_Status]` section, or we could do something more elaborate if we wanted to be able to configure the field list independently in the status display vs. the holdings tab (which feels to me like it might be a good idea, though it adds a bit more complexity). What do you think?

Reply:
Much like VUFIND-1701, this feels like something that should just be there since: a) the display isn't nearly as useful without it; and b) a bunch of my libraries are asking for it. But I was convinced quite handily before that this should be a configurable option. So I'm even more not sure whether this display should be on-off or field-by-field.

Reply:
I think the value of the feature entirely depends on how the holdings fields are utilized. I agree that in most cases, it seems like including these details would be useful/interesting, especially since this only impacts the non-default "show full status" display (and thus won't impact a majority of users by default). But if an institution has really verbose notes, I could imagine a situation where they would want to show them on the record page but not clutter up the search results with them, so it seems to me that there's some value to being able to independently configure the display in each context.

Reply:
> I could imagine a situation where they would want to show them on the record page but not clutter up the search results with them

This would include situations where the results screen is a combined (bento) search and the space used for each result should be minimal.

Reply:
Yep, this sounds good.  I'm happy to try the more elaborate/more flexible option.  Attempting to drive this by writing the usage example first :):

```ini
[Item_Status]

...

; If set to true, the item status display will include additional holdings text fields.
; This option will only take effect when show_full_status is also set to true.
include_holdings_text_fields = true

; The set of additional holdings text fields shown in the item status display. If not
; set explicitly, the default is to show all fields defined in holdings_text_fields[].
;displayed_holdings_text_fields[] = 'holdings_notes'
;displayed_holdings_text_fields[] = 'summary'
```

(better naming suggestions warmly welcomed!)

If that looks OK, I'll gate the new code behind the new boolean and look to `displayed_holdings_text_fields` for the list of fields, falling back to the original call to `holdingsTextFieldNames` if it's not available.

Reply:
Yes, that's pretty much exactly what I had in mind! Thanks.